src/ngraph_encapsulate_op.cc

```diff
@@ -618,10 +629,20 @@ class NGraphEncapsulateOp : public OpKernel {
       void* src_ptr = (void*)DMAHelper::base(&ctx->input(i));
       m_freshness_tracker->MarkFresh(src_ptr, ng_function);
     }
+    int time_copy_tensors_to_host = copy_tensors_to_host.ElapsedInMS();
 
     NGRAPH_VLOG(4)
         << "NGraphEncapsulateOp::Compute done marking fresh for cluster "
         << m_ngraph_cluster;
+    NGRAPH_VLOG(1) << "NGRAPH_TF_TIMING_PROFILE: OP_ID: " << my_instance_id
+                   << " Step_ID: " << ctx->step_id()
+                   << " Cluster: " << ctx->op_kernel().name()
+                   << " Time-Compute: " << compute_time.ElapsedInMS()
+                   << " Function-Create-or-Lookup: "
+                   << time_func_create_or_lookup
+                   << " Create-tensors: " << time_create_or_lookup_tensors
+                   << " Execute: " << time_execute_function
+                   << " Copy-to-host: " << time_copy_tensors_to_host;
```

Comment:
Could we change the log to?
"Create tensors + Copy inputs to Device"
"Copy-Outputs-to-host"